build/document-extractor.js

```diff
@@ -163,19 +176,20 @@ function addSections($) {
         .eq(0);
 
       // Loop over each and every "root element" in the node and keep piling
-      // them up in a buffer, until you encounter a `div.bc-table` then
+      // them up in a buffer, until you encounter a `div.bc-data` or `div.bc-specs` then
       // add that to the stack, clear and repeat.
       const iterable = [...$[0].childNodes];
       let c = 0;
-      let countBCDDataDivsFound = 0;
+      let countSpecialDivsFound = 0;
       iterable.forEach((child) => {
         if (
           child.tagName === "div" &&
           child.attribs &&
           child.attribs.class &&
-          /bc-data/.test(child.attribs.class)
+          (child.attribs.class.includes("bc-data") ||
+            child.attribs.class.includes("bc-specs"))
```

Comment:
If you happen to iterate on this, I'd move the condition to a function with a speaking name, that accepts a `child` and returns a boolean.

I bet, this will make the code more readable.